src/pages/scheduler/components/Toolbar.tsx

```diff
@@ -14,11 +14,27 @@ export const CalendarToolBar =
         onSelectedDateChange(new Date());
       }
     };
+
+    const handleDownload = () => {
+      if (vCalendar) {
+        const element = document.createElement('a');
+        const file = new Blob([vCalendar], { type: 'text/plain' });
+        element.href = URL.createObjectURL(file);
+        element.download = 'calendar.ics';
```

Comment:
small qol change: the filename could be dynamic cause people will probably be downloading multiple schedules. Maybe just like `${term}_calendar.ics` or even use the readable term.

Reply:
Good point. This was me being lazy.